typescript/.changeset/all-nails-cry.md

```diff
@@ -1,5 +1,5 @@
 ---
-"@coinbase/agentkit-vercel-ai-sdk": patch
+"@coinbase/agentkit-vercel-ai-sdk": minor
```

Comment:
This is safe to change. [See here](https://github.com/changesets/changesets/blob/main/docs/common-questions.md#i-want-to-edit-the-summary-or-package-bump-types---is-it-safe-to-do-that)

Reply:
The reason for this change is to publish the first minor version for `@coinbase/agentkit-vercel-ai-sdk` in the next release -> `v0.1.0`.